tools/clang/unittests/HLSL/DxcTestUtils.h

```diff
@@ -125,8 +125,8 @@ class VersionSupportInfo {
   void Initialize(dxc::DxcDllSupport &dllSupport);
   // Return true if IR sensitive test should be skipped, and log comment
   bool SkipIRSensitiveTest();
-  // Return true if DXIL 1.1 test should be skipped, and log comment
-  bool SkipDxil_1_1_Test();
+  // Return true if DXIL with a given version should be skipped, and log comment
```

Comment:
"Return true if test requiring DXIL of given version should be skipped, and log comment"